apps/vr-tests/src/utilities/TestWrapperDecorator.tsx

```diff
@@ -1,48 +1,66 @@
 import * as React from 'react';
-import { DecoratorFunction } from '@storybook/addons';
-import { ExtendedStoryFnReturnType } from './types';
+import type { DecoratorFn } from '@storybook/react';
 
-export const TestWrapperDecorator: DecoratorFunction<ExtendedStoryFnReturnType> = story => (
+export const TestWrapperDecorator: DecoratorFn = (story, context) => (
   <div style={{ display: 'flex' }}>
-    <div className="testWrapper" style={{ padding: '10px', overflow: 'hidden' }}>
+    <div
+      id="testWrapper"
+      className="testWrapper"
+      style={{ padding: '10px', overflow: 'hidden', ...context.parameters.testWrapperStyle }}
```

Comment:
The main idea is to allow defining decorator once per story file and allow styles customization per story, here is an example:

```
export default {
  decorators: [TestWrapperDecorator], // adds decorator for all stories within this file
} satisfies Meta<typeof Divider>


export const DividerVertical: StoryFn<typeof Divider> = () => <Divider vertical />;
DividerVertical.parameters = {
  testWrapperStyle: { // Here we can add some custom styles that will be used by TestWrapperDecorator only for this specific story
    width: '100px',
    height: '300px',  
  }
};
```

Reply:
But we could get rid of it to reduce the number of diffs

Reply:
Rolled back to initial implementation

Reply:
> Q: there will be no intellisense/TS validation for this right ? because StoryFn<typeof Divider> has no sense about the actual Decorator API

That's correct, no intellisense/TS validation in current implementation. 

I've rolled back this change, so it's not relevant anymore.